data/json/items/armor/arms_armor.json

```diff
@@ -383,29 +383,29 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "pair of scrap arm guards", "str_pl": "pairs of scrap arm guards" },
-    "description": "A pair of arm guards made from scraps of metal secured by simple strings; the loose collection of plates provides decent but not the most convenient protection.",
+    "description": "A pair of arm guards made from scraps of metal secured by simple strings to cloth wraps.",
     "weight": "3064 g",
     "volume": "4750 ml",
     "price": 20000,
     "price_postapoc": 500,
```

Comment:
If I wanted you to change the price listed here, a comment would be the best option. Comments mean you don't need a separate PR, which means there's no worry about conflicting work.